content/en/docs/pluggable-components/network-policy.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Network Policy"
+keywords: "Kubernetes, KubeSphere, NetworkPolicy"
+description: "How to Enable Network Policy"
+
+linkTitle: "Network Policy"
+weight: 3547
+---
+
+## What is Network Policy
+
+Starting from v3.0.0, users can configure network policies of native Kubernetes in KubeSphere. Network Policies are an application-centric construct, enabling you to specify how a pod is allowed to communicate with various network entities over the network. With network policies, users can achieve network isolation within the same cluster, which means firewalls can be set up between certain instances (pods).
+
+{{< notice note >}}
+
+Please make sure that the CNI network plugin used by the cluster supports Network Policies before you enable it. There are a number of CNI network plugins that support Network Policies, including Calico, Cilium, Kube-router, Romana and Weave Net.
+
+{{</ notice >}} 
+
+For more information, see [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+
+## Enable Network Policy before Installation
+
```

Comment:
Add a prerequisite: It's recommended that you use [Calico](https://www.projectcalico.org/) as the CNI before you enable Network Policy.